test/system/160-volumes.bats

```diff
@@ -387,4 +387,27 @@ NeedsChown    | true
     run_podman volume rm $myvolume
 }
 
+@test "podman volume mount" {
+    skip_if_remote "podman --remote volume mount not supported"
+    myvolume=myvol$(random_string)
+    myfile=myfile$(random_string)
+
+    # Create a named volume
+    run_podman volume create $myvolume
+    is "$output" "$myvolume" "output from volume create"
+
+    if ! is_rootless ; then
+        # image mount is hard to test as a rootless user
+        # and does not work remotely
+        run_podman volume mount ${myvolume}
+        mnt=${output}
+	touch $mnt/$myfile
+        run_podman run -v ${myvolume}:/vol1:z $IMAGE ls /vol1
```

Comment:
Whether you accept my `echo/cat` suggestion or not, please run with `--rm` to avoid littering

Reply:
Fixed